### PR TITLE
doc: zebra start command config

### PR DESCRIPTION
**Pull request by danielvladco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,11 +79,7 @@ and for Zebra:
 ```toml
 kind = "zebra"
 path = "path/to/zebra/repo"
-start_command = "cargo +stable r -- --verbose start"
-# cargo +stable r   run Zebra using stable Rust
-# --                all args after this will get passed to Zebra
-# verbose           enables verbose logging
-# start             starts the node
+start_command = "target/release/zebrad --verbose"
```
Comment (niklaslong) [inline README.md:82]: What's the conclusion on `--verbose`, does it make a difference?

**Review by niklaslong — APPROVED** [review]
Other than the pending comment, this looks good to me.

File: README.md (hunk 1 above)
Comment (Rqnsom) [reply README.md:82]: And whether we need to use the `start` option or not.
It's funny behaviour if, with no options provided, zebrad defaults to starting the node (which can be achieved with the `start` option.

Comment (danielvladco) [reply README.md:82]: I don't see any additional logs with `--verbose` when running the tests but maybe it's useful in other scenarios idk. Anyway I can remove it.

Comment (niklaslong) [reply README.md:82]: I'd suggest keeping `start` and removing `--verbose` in that case. Just to be explicit.

**Pull request merged**